src/test/kotlin/org/opensearch/replication/MultiClusterRestTestCase.kt

```diff
@@ -421,6 +422,29 @@ abstract class MultiClusterRestTestCase : OpenSearchTestCase() {
     }
 
     protected fun wipeIndicesFromCluster(testCluster: TestCluster) {
+
```

Comment:
Minor: We can move this logic to a new method instead. Something like `stopAllReplicationJobs`